app/services/personal_care_needs/importer.rb

```diff
@@ -17,7 +17,6 @@ def call
         personal_care_needs.map { |personal_care_need| build(personal_care_need) },
         ASSESSMENT_ANSWER_CATEGORY
       )
-      profile.save!
```

Comment:
Would it be worth adding a comment here to point out that `merge_assessment_answers` triggers a save? This `save!` statement does appear unnecessary, but without it we have an implicit reliance on `merge_assessment_answers` doing a save. Any ideas on how we could be defensive here?

Reply:
merge_assessment_answers! doesn't trigger a save.